### DISCUSSION TITLE
Get a mail when a sonde lands close

### DISCUSSION THREAD

**Discussion by micked99** [dl9rdz/rdz_ttgo_sonde#387] [TTGO firmware]
Hi all, one feature that would be nice to have in the fw, have the rdz to send a mail whenever a sonde have landed closeby, make this adjustable from a few km out to say 100km or so. 
Tnx  
/ Mikael - SA6BSS

**Comment by DG0MG** [comment]
You can use exact this feature if you signup on radiosondy.info. But it doesn't just depend on only *YOUR* TTGO, but on all receiving stations, which is MUCH better. The TTGO should only be a "data collector", other functions should be implemented in the larger frontend.

**Comment by micked99** [comment]
Hi, I went and looked at sondyinfo and my email requst was inactive so I re-enabled it again, so all fine now hopefully  👍

/Mikael
